### ISSUE TITLE
Super Scaffolded attachments aren't being serialized correctly

### ISSUE THREAD

**Issue by gazayas** [bullet-train-co/bullet_train#310]
To duplicate:
```
> rails g model Foo team:references bar:attachment
> bin/super-scaffold crud Foo Team bar:file_field
> bundle exec rails test
```

The tests were working before in https://github.com/bullet-train-co/bullet_train-super_scaffolding/commit/5682c4bc7251552ae7bf7f38b1100158d787f001, but they're not passing now and I can't seem to find where the change has been made. I don't think it was the most recent change in `bullet_train-super_scaffolding`. I'm opening the issue here because I'm not sure if it was due to a change over there or in another package.

Basically our json response for attachments is returning values like this:
```
"bar"=>"#<ActiveStorage::Attached::One:0x00007f53d6494130>"
```

Will keep investigating.

**Comment by gazayas** [comment]
Haven't quite figured this one out yet, but I'm thinking we'll need something like this in our serializers. However, the `file_field_value` method below isn't responding.

I wonder if it's because we're using `jsonapi-serializer` as opposed to `ActiveModel::Serialzer`.

```ruby
class Api::V1::Scaffolding::CompletelyConcrete::TangibleThingSerializer < Api::V1::ApplicationSerializer
  set_type "scaffolding/completely_concrete/tangible_thing"
  include Rails.application.routes.url_helpers

  attributes :id,
    ...
    :file_field_value,
    ...

  def file_field_value
    rails_blob_path(object.file_field_value, disposition: "attachment", only_path: true) if object.file_field_value.attached?
  end

  belongs_to :absolutely_abstract_creative_concept, serializer: Api::V1::Scaffolding::AbsolutelyAbstract::CreativeConceptSerializer
end
```

Here are a few articles that mention serializing attachments this way:
[Article 1](https://medium.com/swlh/rails-serializing-active-storage-uploads-with-active-model-9bf543ed25eb)
[Article 2](https://medium.com/swlh/active-storage-with-rails-api-23d6a72ed3f2)
[Article 3](https://medium.com/@elenichappen/using-active-storage-with-active-model-serializers-36c91f5a0686)

**Comment by gazayas** [comment]
https://github.com/jsonapi-serializer/jsonapi-serializer/issues/131

**Comment by gazayas** [comment]
Okay, got this to work with `rails_blob_path`, will try to push some tests sometime soon.
